docs/standard/commandline/syntax.md

```diff
@@ -0,0 +1,577 @@
+
+In some command-line tools, a difference in casing specifies a difference in function. For example, [`git clean -X`](https://git-scm.com/docs/git-clean#Documentation/git-clean.txt--X) behaves differently than [`git clean -x`](https://git-scm.com/docs/git-clean#Documentation/git-clean.txt--x). The .NET CLI is all lowercase.
+
+## The `--` token
+
+POSIX convention interprets the double-dash (`--`) token as an escape mechanism. Everything that follows the double-dash token is interpreted as arguments for the command. This functionality can be used to submit arguments that look like options, since it prevents them from being interpreted as options.
+
+Suppose *myapp* takes a `message` argument, and you want the value of `message` to be `--interactive`. The following command line might give unexpected results.
+
+```console
+myapp --interactive
+```
+
+If `myapp` doesn't have an `--interactive` option, the `--interactive` token is interpreted as an argument. But if the app does have an `--interactive` option, this input will be interpreted as referring to that option.
+
+The following command line uses the double-dash token to set the value of the `message` argument to "--interactive":
+
+```console
+myapp -- --interactive
+      ^^
+```
+
+`System.CommandLine` supports this double-dash functionality.
+
+Several commands in the .NET CLI have a special implementation of the double-dash token. In the case of `dotnet run`, `dotnet watch`, and `dotnet tool run`, tokens that follow `--` are passed to the app that is being run by the command. For example:
+
+```dotnetcli
+dotnet run --project ./myapp.csproj -- --message "Hello world!"
+                                    ^^
+```
+
+In this example, the `--project` option is passed to the `dotnet run` command, and the `--message` option with its argument is passed as a command-line option to *myapp* when it runs.
+
+The `--` token is not always required for passing options to an app that you run by using `dotnet run`. Without the double-dash, the `dotnet run` command automatically passes on to the app being run any options that aren't recognized as applying to `dotnet run` itself or to MSBuild. So the following command lines are equivalent because `dotnet run` doesn't recognize the arguments and options:
+
+```dotnetcli
+dotnet run -- quotes read --delay 0 --fg-color red
+dotnet run quotes read --delay 0 --fg-color red
+```
+
+This behavior doesn't follow POSIX standards, and `System.CommandLine` doesn't have built-in support for it.
+
+## Option-argument delimiters
+
+`System.CommandLine` lets you use a space, '=', or ':' as the delimiter between an option name and its argument. For example, the following commands are equivalent:
+
+```dotnetcli
+dotnet build -v quiet
+dotnet build -v=quiet
+dotnet build -v:quiet
+```
+
+A POSIX convention lets you omit the delimiter when you are specifying a single-character option alias. For example, the following commands are equivalent:
+
+```console
+myapp -vquiet
```

Comment:
Oh, that is truly awful @jonsequitur But I can live iwth it if it is Posix. Can we have a feature request that authors can turn off this horribleness